snooty/types.py

```diff
@@ -96,6 +96,21 @@ def __load(self) -> None:
             self._checksum = hashlib.blake2b(self._data, digest_size=32).hexdigest()
 
 
+@checked
+@dataclass
+class BannerInstantiated:
```

Comment:
I don't like this name, but also haven't thought of a good alternative.

Reply:
How about `ParsedBannerConfig`?